local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -94,6 +98,16 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 	return nil
 }
 
+// GetFsAvailable returns available capacity about a mounted filesystem. fullPath
+// is the pathname of any file within the mounted filesystem.
+func (u *volumeUtil) GetFsAvailable(fullPath string) (uint64, error) {
+	var s syscall.Statfs_t
+	if err := syscall.Statfs(fullPath, &s); err != nil {
+		return 0, err
+	}
+	return uint64(s.Frsize) * s.Bavail, nil
```

Comment:
@msau42 I'm ok with using total blocks for reasons listed in ur comment (pre-populated data and consistent capacity).  My concern is that the total block is generally not the capacity user can actually use, e.g. for ext4, there's 5% system reserved space.